docs/reference/module-options.md

```diff
@@ -372,3 +372,442 @@ module.exports = {
 }
 ```
 
+## Options for the core page module
+
+Option settings in this section apply to the core page module (`@apostrophecms/page`).
+
+| Option | Value type | Description |
+|---------|---------|---------|
+| [`builders`](#builders) | Object | Set query builder values to be used when pages are served. |
+| [`home`](#home) | Boolean/Object | Change how the home page is added to `req.data` when pages are served. |
+| [`minimumPark`](#minimumpark) | Array | Override home page default properties when created. |
```

Comment:
This description is meant to be short and incomplete. Since the trash page is registered as a child of the home page I phrased it like this. The trash page is included in the full documentation below.

Reply:
I see, but it strongly implies it's only about the home page which isn't the case. How about:

"Override default parked pages, such as the home page."